evolver/device.py

```diff
@@ -42,7 +42,13 @@ def effectors(self):
 
     @property
     def calibration_status(self):
-        return {name: device.calibrator.status for name, device in self.hardware.items()}
+        """Return the calibration Status for each device's calibrator. This is explicit and returns all available
```

Comment:
perhaps `Status for each device's calibrator.` -> `Status for each device's calibrator, or None if hardware has no calibrator.`